src/main/scala/nl.knaw.dans.easy.bagstore/ItemId.scala

```diff
@@ -33,12 +33,20 @@ abstract class ItemId(val uuid: UUID) {
 }
 
 object ItemId {
+
   def fromString(s: String): Try[ItemId] = Try {
     s.split("/", 2) match {
-      case Array(uuidStr) => BagId(UUID.fromString(uuidStr))
+      case Array(uuidStr) => BagId(UUID.fromString(validateUuidLength(uuidStr)))
       case Array(uuidStr, path) =>
-        FileId(UUID.fromString(uuidStr), Paths.get(URLDecoder.decode(path, "UTF-8")))
+        FileId(UUID.fromString(validateUuidLength(uuidStr)), Paths.get(URLDecoder.decode(path, "UTF-8")))
+    }
+  }
+
+  private def validateUuidLength(uuidAsString: String): String = {
+        if (uuidAsString.length > 36) {
```

Comment:
I think we should also refuse any UUIDs that are too short. Perhaps it is allowed, strictly speaking, but when a UUID such as 1-2-3-4-5 is specified it is probably a mistake. Let's err and the safe side and reject it.

Reply:
done